packages/plugins/persisted-operations/src/index.ts

```diff
@@ -1,25 +1,16 @@
-import { Plugin, PromiseOrValue } from 'graphql-yoga'
+import { GraphQLParams, Plugin, PromiseOrValue } from 'graphql-yoga'
 import { GraphQLError } from 'graphql'
 
-export interface PersistedOperationsStore {
-  get(key: string): PromiseOrValue<string | null | undefined>
-}
-
-export interface PersistedOperationExtension {
-  version: 1
-  sha256Hash: string
-}
-
-function decodePersistedOperationsExtension(
-  input: Record<string, any> | null | undefined,
-): null | PersistedOperationExtension {
+function defaultGetPersistedOperationKey(params: GraphQLParams): null | string {
   if (
-    input != null &&
-    typeof input === 'object' &&
-    input?.version === 1 &&
-    typeof input?.sha256Hash === 'string'
+    params.extensions != null &&
+    typeof params.extensions === 'object' &&
+    params.extensions?.persistedQuery != null &&
+    typeof params.extensions?.persistedQuery === 'object' &&
+    params.extensions?.persistedQuery.version === 1 &&
+    typeof params.extensions?.persistedQuery.sha256Hash === 'string'
   ) {
-    return input as PersistedOperationExtension
+    return params.extensions?.persistedQuery.sha256Hash
   }
```

Comment:
should we reconsider the default and use https://github.com/graphql/graphql-over-http/issues/216#issuecomment-1217469059 as proposed by @benjie? 🤔  I think it might be pretty close or maybe even the final naming in the possible GraphQL over HTTP spec.

Reply:
URQL and Apollo still support this spec only(not APQ, I'm referring to `forcePersistedQueries` mode of these).
https://formidable.com/open-source/urql/docs/advanced/persistence-and-uploads/#customizing-hashing